src/main/java/org/opensearch/knn/indices/Model.java

```diff
@@ -69,13 +88,13 @@ public boolean equals(Object obj) {
 
         EqualsBuilder equalsBuilder = new EqualsBuilder();
         equalsBuilder.append(modelMetadata, other.modelMetadata);
-        equalsBuilder.append(modelBlob, other.modelBlob);
+        equalsBuilder.append(modelBlob.get(), other.modelBlob.get());
```

Comment:
how are we accessing modelBlob as variable? shouldn't this be getModelBlolb?

Reply:
Will update to use getter